### DISCUSSION TITLE
Text Find and Replace question

### DISCUSSION THREAD

**Discussion by zachzachry** [WASasquatch/was-node-suite-comfyui#193] [Q&A]
What do I put in the Find field if I want to search for a word?  I am using [/title/]  and every instance of t,i,l, and e are being replaced instead of the word.

**Comment by WASasquatch** [comment]
> [/title/]

If you just want to match "title" just use `title` in find.

**Comment by zachzachry** [comment]
Thanks so much.  Nothing better than spending a few hours over thinking.

**Reply by WASasquatch** [reply]
Tell me about it :(
